ephios/extra/templatetags/rich_text.py

```diff
@@ -15,8 +15,6 @@
     "code",
     "div",
     "em",
-    "h1",
-    "h2",
```

Comment:
Well actually we want these headings for pages.

Reply:
Can we make not having them optional?

Reply:
do we? as the title is already rendered as h1 i'd prefer to not have h1 in the content there.

Reply:
I'd opt for having them. Not having them would make `rich_text` really unintuitive.